### PR TITLE
Remove malformed check for already used ports (fixes #147)

### LINKED ISSUE #147: gretty 3.0.2 java.net.ConnectException: Connection refused
I have already tried version 3.0.2 but there's error about connection refused. is there any new config?
Edit1: I am using gradle 6.3-rc1
Edit2: I already tried gradle 6.3 but still error

```
Caused by: java.net.ConnectException: Connection refused: connect
        at org.akhikhl.gretty.ServiceProtocol$Writer.write(ServiceProtocol.groovy:76)
        at org.akhikhl.gretty.ServiceProtocol$Writer$write.call(Unknown Source)
        at org.akhikhl.gretty.LauncherBase$_beforeLaunch_closure2.doCall(LauncherBase.groovy:71)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at org.akhikhl.gretty.LauncherBase.beforeLaunch(LauncherBase.groovy:69)
        at org.akhikhl.gretty.DefaultLauncher.super$2$beforeLaunch(DefaultLauncher.groovy)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at org.akhikhl.gretty.DefaultLauncher.beforeLaunch(DefaultLauncher.groovy:58)
        at org.akhikhl.gretty.LauncherBase.launch(LauncherBase.groovy:155)
        at org.akhikhl.gretty.Launcher$launch.call(Unknown Source)
        at org.akhikhl.gretty.StartBaseTask.action(StartBaseTask.groovy:79)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:104)
```

### PR DESCRIPTION
**Pull request by f4lco**
Because #138 removed adequate error handling, the check for
already used ports was broken and caused Gretty to fail in
case the 'gretty_ports.properties' file was present.

I think the check is now obsolete because #138 removed the
ability to manually specify fixed service and status ports
for Gretty. As a result, Gretty's ports will never clash
from one Gretty invocation to the next.

If the check was restored, it would only check if the
previous Gretty invocation had already terminated.

If one of the *servlet container ports* clashed, I deem
the current behavior (BindException) sufficient.

---
tl;dr: I tried to wrap my head around the question why would anybody check the ports of the prior execution and could not find a good answer. Maybe the check made sense when we still could tell Gretty to use fixed service/status ports (which is no longer the case). If you have an idea or any concerns, please share them.

### CODE REVIEW

**Comment by boris-petrov** [comment]
I'm not sure I completely follow the reasoning here but I guess it doesn't hurt to try it if it fixes the people's issues. We can merge, release and see. If there are more problems, we could try resolving them or just revert everything.

@javabrett - if you agree, you can merge this and release.

@f4lco - is there some kind of test that could be written for the thing that broke?

**Comment by f4lco** [comment]
@boris-petrov "reasoning", indeed. I simply cannot find a reason why this check (with this precise implementation) was useful in the first place, and tried to give a potential explanation. If there are any questions, I'd be glad to answer.
No, I can't think of a test case for that. A fragile check using the port information of the previous (in most cases terminated) Gretty invocation has been removed. That would leave us with one thing less to test.
The check tried to find out if the server was already running. If that is the case, and the user demanded to start two servlet containers with identical port information, a `BindException` is inevitable. I think that's informative to the user and is not connected to Gretty in particular.

**Comment by f4lco** [comment]
Dear @javabrett a couple of weeks have passed. Is there any update on your side? I have rebased this PR and https://github.com/gretty-gradle-plugin/gretty/pull/153 to integrate recent commits to master.

**Comment by javabrett** [comment]
@f4lco I hope to run a release this week incorporating this fix.  Thanks.
